mod/metric_answer_lookup/set/type/metric_value.rb

```diff
@@ -1,3 +1,3 @@
 event :update_answer_lookup_table_due_to_answer_change, :finalize, on: :update do
-  update_answer answer_id: id
+  hybrid? ? update_answer(id: answer.id) : update_answer(answer_id: id)
```

Comment:
`update_answer(id: answer.id)`   is this just to tick updated_at or something?

Reply:
since calculated answers don't have an answer_id we can't find them using the answer_id

Reply:
oh, I misread this and was thinking we were updating the attribute.  doh.